mlflow/pyfunc/__init__.py

```diff
@@ -460,7 +460,7 @@ def predict(*args):
 
 
 def save_model(path, loader_module=None, data_path=None, code_path=None, conda_env=None,
-               model=Model(), python_model=None, artifacts=None):
+               model=Model(), python_model=None, artifacts=None, **kwargs):
     """
```

Comment:
We can avoid documenting **kwargs in our Sphinx API docs by renaming the method prototype in the first line of the docstring, e.g:

```
def save_model(path, loader_module=None, data_path=None, code_path=None, conda_env=None,
               model=Model(), python_model=None, artifacts=None, **kwargs):
    """
    save_model(path, loader_module=None, data_path=None, code_path=None, conda_env=None,\
               model=Model(), python_model=None, artifacts=None)

    Create a custom Pyfunc model, incorporating custom inference logic and data dependencies.
    ...
    """
```

I think we should do this because the `kwargs` parameter only serves the purpose of providing an alias for another keyword argument that is explicitly named in the method prototype.